src/main/java/frc/robot/commands/SetShooterHoodEdge.java

```diff
@@ -0,0 +1,33 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.ShooterHood;
+import frc.robot.util.PressureSensor;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/latest/docs/software/commandbased/convenience-features.html
+public class SetShooterHoodEdge extends SequentialCommandGroup {
+  /**
+   * Creates a new SetShooterHoodEdge.
+   * 
+   * @param shooterHood    Shooter hood subsystem
+   * @param pressureSensor Pressure sensor
+   * @param position       Top or bottom? (true=top, false=bottom)
+   */
+  public SetShooterHoodEdge(ShooterHood shooterHood, PressureSensor pressureSensor, boolean position) {
```

Comment:
```suggestion
  public SetShooterHoodEdge(ShooterHood shooterHood, PressureSensor pressureSensor, boolean top) {
```
Boolean variables are clearer when they describe what true means.